Refresh.GameServer/Types/Assets/GameAssetType.cs

```diff
@@ -98,4 +98,8 @@ public enum GameAssetType
     /// Magic: PTGb
     /// </remarks>
     Painting = 12,
+    /// <summary>
+    /// An image, stored in a TGA container.
+    /// </summary>
```

Comment:
Should leave remarks about how there technically is no magic, add a seealso to point to the function that detects it